src/databricks/labs/ucx/queries/views/direct_fs_access.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+    *
```

Comment:
nit: as a good practice of views, please specify explicit column names

Reply:
… especially for `UNION` where the order of columns is important. 

(People often assume that schema changes which modify the column order are forward and backward compatible and don't realise `UNION`s will break. The only solution is to defensively enumerate columns explicitly when performing a `UNION`.)

Reply:
dropped from this PR